FWCore/Services/plugins/CondorStatusUpdater.cc

```diff
@@ -209,8 +209,9 @@ void CondorStatusService::beginPost() {
     ss_max_lumis << maxLumis;
     updateChirp("MaxLumis", ss_max_lumis.str());
   }
-
-  m_beginJob = std::chrono::system_clock::to_time_t(std::chrono::system_clock::now());
+  if (m_beginJob == 0.) {
+    m_beginJob = std::chrono::system_clock::to_time_t(std::chrono::system_clock::now());
+  }
```

Comment:
Is setting `m_beginJob` in `beginPost()` still useful, given that the `firstUpdate()` is called from the constructor even before the call to `beginPost()` is registered in ActivityRegistry for postBeginJob signal?

Reply:
It is possible that the CondorStatusUpdater is used outside of cmsRun and the TimerServiceBase was not initialized so the value of the call would be 0. But given we are trying to capture the time for more of the application, moving the fallback to the constructor would probably make more sense.